src/efr.jl

```diff
@@ -79,6 +69,17 @@ function differential_filter!(u, setup, filter_radius, relax_parameter)
     return u
 end
 
+function decompose_filter_mat(setup, filter_radius)
+	(; dimension, Nu) = setup.grid
+	D = dimension()
+    ntuple(D) do α
+		L = diffusion_mat_velocity(setup, α)
+		Id = sparse(I, prod(Nu[α]), prod(Nu[α]))
+		filter_mat = Id - 2 * (filter_radius^2) * L
+		lu(filter_mat)
```

Comment:
With this syntax the matrix is sparse, and the `lu` will use a sparse factorization via SuiteSparse.
This reduces the computations for larger grids.